src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CcdCallbackController.java

```diff
@@ -48,6 +54,28 @@ public CallbackResponse attachToCase(
         }
     }
 
+    @PostMapping(path = "/create-case")
+    public CallbackResponse createCase(
```

Comment:
Because we're adding this endpoint, we should un-ignore functional tests for it as part of this PR.

Reply:
CCD API not yet in. Introduced controller first as it completes integration testing efficiently without mocking

Reply:
I think we should wait with merging this change until the CCD API part is done.

Reply:
OK. Even though I agree and normally I would develop as we wish but it is easier to build up-to-standards testing framework against source code with integration testing once endpoint is in. Rather than doing CCD part and mocking lot's of things where mocks are already in place and unit test is 600+ lines already.

+! endpoint is not available anyway: no definition file points to it and even if you try - service configuration does not allow it - no transformation endpoint configured. Fail fast approach.

I know this is breaking the mindset, but really - faster approach to accomplishment